app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,106 @@
+    <string name="network_videos_sharename_summary">Exemple: / vidéo / aérien</string>
+    <string name="network_videos_sharename_title">"Nom de l'approbation du réseau"</string>
+    <string name="network_videos_smb_dialects_summary">Sélectionnez la version du protocole SMB</string>
+    <string name="network_videos_smb_dialects_title">Sélectionnez le dialecte du protocole SMB</string>
+    <string name="network_videos_test_connection">Tester les paramètres de connexion</string>
+    <string name="network_videos_username_summary">Optionnelle</string>
+    <string name="network_videos_username_title">"Nom d'utilisateur"</string>
+
+    <!-- Settings -->
+    <string name="capabilities_codecs_title">Prise en charge des codecs</string>
+    <string name="capabilities_decoders_title">Décodeur disponible</string>
+    <string name="capabilities_display_title">TV / Affichage</string>
+    <string name="capabilities_resolution_title">"Résolution d'écran"</string>
+
+    <string name="interface_alt_text_position_summary">"Modifiez la position de l'horloge et de l'emplacement après chaque vidéo pour éviter les brûlures (OLED)"</string>
+    <string name="interface_alt_text_position_title">Position de texte alternative</string>
+    <string name="interface_enable_skip_videos_summary">Sauter une vidéo avec la télécommande (droite / gauche à D-Pad)</string>
+    <string name="interface_enable_skip_videos_title">Vidéo suivante / précédente</string>
+    <string name="interface_mute_videos_summary">Lecture des vidéos sans le son</string>
+    <string name="interface_mute_videos_title">Mute</string>
+    <string name="interface_remove_duplicates_summary">Supprimer les vidéos avec le même nom de fichier de la playlist</string>
+    <string name="interface_remove_duplicates_title">Supprimer des vidéos dupliquées</string>
+    <string name="interface_show_clock_summary">"Montre l'heure actuelle"</string>
+    <string name="interface_show_clock_title">Horloge de spectacle</string>
+    <string name="interface_show_location_style_summary">Choisissez le style du titre vidéo</string>
+    <string name="interface_show_location_style_title">Vidéos Apple</string>
+    <string name="interface_show_location_summary">Affiche un titre ou un emplacement de la vidéo en cours</string>
+    <string name="interface_show_location_title">Afficher les titres vidéo</string>
+    <string name="interface_shuffle_videos_summary">Lecture des vidéos dans un ordre aléatoire</string>
+    <string name="interface_shuffle_videos_title">"Mélangez l'ordre des vidéos"</string>
+    <string name="interface_summary">Customisation du look</string>
+    <string name="interface_title">Configuration de la mise en page</string>
+
+    <string name="performance_buffering_strategy_summary">Changer les priorités avec le stockage intermédiaire peut aider avec des connexions instables</string>
+    <string name="performance_buffering_strategy_title">Stratégie de tampon</string>
```

Comment:
That's correct, but I'd go with "Stratégie de préchargement" (Preloading strategy) because tampon (buffer) can't really be used out of context in French